Makefile

```
@@ -26,7 +34,7 @@ build: _build _build-frontend _package
 # Build the Go project.
 _build:
 	mkdir -p $(BUILD_DIR) && \
-	go build -o $(BUILD_DIR)/$(BINARY_NAME) ./cmd/server
+	go build -C $(BACKEND_BASE_DIR) -o ../$(BUILD_DIR)/$(BINARY_NAME) ./cmd/server
```

```suggestion
	go build -C $(BACKEND_BASE_DIR) -o ../$(BUILD_DIR)/$(BINARY_NAME) ./cmd/server
```

Do we need `..` in `-o` flag?

---

We need that. o/w `target` directory is created in `BACKEND_BASE_DIR`